aspnetcore/security/authorization/simple.md

```diff
@@ -69,15 +69,36 @@ This would allow only authenticated users to the `AccountController`, except for
 > [!WARNING]
 > `[AllowAnonymous]` bypasses all authorization statements. If you combine `[AllowAnonymous]` and any `[Authorize]` attribute, the `[Authorize]` attributes are ignored. For example if you apply `[AllowAnonymous]` at the controller level, any `[Authorize]` attributes on the same controller (or on any action within it) is ignored.
 
+The following code limits access to the `LoginModel` Razor Page to authenticated users:
+
+```csharp
+[Authorize]
+public class LoginModel : PageModel
```

Comment:
Should we call this `LogOutModel`? Feels really weird to have authorization on a page that let's you login.